src/main/scala/cromwell/engine/backend/jes/JesBackendCall.scala

```diff
@@ -112,4 +114,11 @@ class JesBackendCall(val backend: JesBackend,
 
   override def useCachedCall(avoidedTo: BackendCall)(implicit ec: ExecutionContext): Future[ExecutionHandle] =
     backend.useCachedCall(avoidedTo.asInstanceOf[JesBackendCall], this)
+
+  override val pollBackoff = new ExponentialBackOff.Builder()
+    .setInitialIntervalMillis(30.seconds.toMillis.toInt)
+    .setMaxIntervalMillis(2.minutes.toMillis.toInt)
```

Comment:
Yeah - I was even wondering if starting at something like 3' (or whatever the minimum JES runtime is + some seconds of margin) would make sense, because we know that polling before is going to return "not finished yet" anyway. But then you start with 3' cycles which is not cool if your job takes 3'05"..